photutils/02_photutils_source_detection.ipynb

```diff
@@ -0,0 +1,1034 @@
+    "\n",
+    "If we set all these parameters to be equivalent, though, we should find much better agreement between the two methods:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "iraffind_match = IRAFStarFinder(fwhm=5.0, threshold=20.*std, \n",
+    "                                sharplo=0.2, sharphi=1.0, \n",
+    "                                roundlo=-1.0, roundhi=1.0,\n",
+    "                                minsep_fwhm=0.0)\n",
+    "sources_iraf_match = iraffind_match(xdf_image * ~xdf_image.mask)    \n",
+    "print(sources_iraf_match)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The number of detected sources are in much better agreement now - 1415 versus 1470 - but the improved agreement can also be seen by plotting the location of these sources:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Set up the figure with subplots\n",
+    "fig, [ax1, ax2] = plt.subplots(1, 2, figsize=(12, 6))\n",
+    "plt.tight_layout()\n",
+    "\n",
+    "# Plot the DAOStarFinder data\n",
+    "fitsplot = ax1.imshow(np.ma.masked_where(xdf_image.mask, xdf_image_clipped), norm=norm_image)\n",
+    "ax1.scatter(sources_dao['xcentroid'], sources_dao['ycentroid'], s=30, marker='o', \n",
+    "            lw=1, alpha=0.7, facecolor='None', edgecolor='r')\n",
+    "ax1.set_xlabel('X (pixels)')\n",
+    "ax1.set_ylabel('Y (pixels)')\n",
+    "ax1.set_title('DAOStarFinder Sources')\n",
+    "\n",
+    "# Plot the IRAFStarFinder data\n",
+    "fitsplot = ax2.imshow(np.ma.masked_where(xdf_image.mask, xdf_image_clipped), norm=norm_image)\n",
+    "ax2.scatter(sources_iraf_match['xcentroid'], sources_iraf_match['ycentroid'], \n",
+    "            s=30, marker='o', lw=1, alpha=0.7, facecolor='None', edgecolor='r')\n",
+    "ax2.set_xlabel('X (pixels)')\n",
+    "ax2.set_title('IRAFStarFinder Sources')\n",
+    "\n",
+    "# Define the colorbar\n",
+    "cbar_ax = fig.add_axes([1, 0.09, 0.03, 0.87])\n",
+    "cbar = plt.colorbar(fitsplot, cbar_ax, ticks=LogLocator(subs=range(10)))\n",
+    "labels = ['$10^{-4}$'] + [''] * 8 + ['$10^{-3}$'] + [''] * 8 + ['$10^{-2}$']\n",
+    "cbar.ax.set_yticklabels(labels)\n",
+    "cbar.set_label(r'Flux Count Rate ($e^{-1}/s$)', rotation=270, labelpad=30)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Take this example as reminder to be critical when selecting a source detection algorithm, and when defining algorithm parameters!"
```

Comment:
I'm a bit confused by this wording. Is it "critical" as in "pay attention to the details"?  If so, maybe a better option would be "reminder to be careful" or even "examine your results and play with the parameters" or something like that?